shoes-core/lib/shoes/builtin_methods.rb

```diff
@@ -37,6 +37,13 @@ def error(message = '')
       Shoes.logger.error message
     end
 
+    def exit(message = '')
+      Shoes::LOG << ['exit', message]
```

Comment:
From other methods :) If someone would like to pass a message before exit, why not? Then it would be better to use "puts" and display it on command line instead of "Shoes::LOG << ['exit', message]" . What do you think? :)